crates/anvil-polkadot/tests/it/mining.rs

```diff
@@ -32,22 +32,21 @@ async fn test_invalid_mining() {
         .unwrap(),
         None
     );
-
     assert!(matches!(
         node.eth_rpc(EthRequest::Mine(Some(U256::from(u128::MAX)), None)).await.unwrap(),
         ResponseResult::Error(RpcError {
-            code: ErrorCode::InvalidParams,
+            code: ErrorCode::InternalError,
             message,
             data: None
-        }) if message == "The number of blocks is too large"
+        }) if message == "Invalid params: The number of blocks is too large"
     ));
     assert!(matches!(
         node.eth_rpc(EthRequest::Mine(None, Some(U256::from(u128::MAX)))).await.unwrap(),
         ResponseResult::Error(RpcError {
-            code: ErrorCode::InvalidParams,
+            code: ErrorCode::InternalError,
             message,
             data: None
-        }) if message == "The interval between blocks is too large"
+        }) if message == "Invalid params: The interval between blocks is too large"
```

Comment:
commented in other places as well about this change. Can't tell easily why we want it